third_party/nvfuser/csrc/python_frontend/python_bindings.cpp

```diff
@@ -77,6 +77,26 @@ void initNvFuserPythonBindings(PyObject* module) {
     ss << "Tensor(index=" << self.index << ", dims=" << self.dims << ")";
     return ss.str();
   });
+  tensor_class.def("__add__", [](nvfuser::Tensor& self) {});
+
+  tensor_class.def(
+      "get_fusion_definition", [](nvfuser::Tensor& self) { return self.fd; });
```

Comment:
Looks like an API that's simply used for debugging, maybe `_get_fusion_definition` instead.

Reply:
Yeah we can hide it. Although it might also be useful in user code, for example if you wanted to write a library of functions to use when building fusions.